angular/projects/spark-angular/src/lib/directives/sprk-box/sprk-box.stories.ts

```diff
@@ -0,0 +1,46 @@
+import { storyWrapper } from '../../../../../../.storybook/helpers/storyWrapper';
+import { SprkBoxDirective } from './sprk-box.directive';
+import { SprkBoxModule } from './sprk-box.module';
+import { markdownDocumentationLinkBuilder } from '../../../../../../../storybook-utilities/markdownDocumentationLinkBuilder';
+
+export default {
+  title: "Components/Box",
+  component: SprkBoxDirective,
+  decorators: [storyWrapper(storyContent => `<div class="sprk-o-Box sb-box-story">${storyContent}<div>`)],
```

Comment:
You can use the `sb-decorate` class here instead of adding a new class of sb-box-story.

Reply:
`sb-decorate` has a height that I didn't want, that's why I made this